receiver/awscontainerinsightreceiver/internal/stores/podstore.go

```diff
@@ -472,13 +472,19 @@ func (p *PodStore) addStatus(metric CIMetric, pod *corev1.Pod) {
 					switch {
 					case containerStatus.State.Running != nil:
 						metric.AddTag(ci.ContainerStatus, "Running")
+						metric.AddField(ci.ContainerStatusRunning, 1)
 					case containerStatus.State.Waiting != nil:
 						metric.AddTag(ci.ContainerStatus, "Waiting")
+						metric.AddField(ci.ContainerStatusWaiting, 1)
 						if containerStatus.State.Waiting.Reason != "" {
 							metric.AddTag(ci.ContainerStatusReason, containerStatus.State.Waiting.Reason)
+							if strings.Contains(containerStatus.State.Waiting.Reason, "Crash") {
```

Comment:
This is the most iffy part of the code review, IMO. I couldn't find an enum or anything to determine what "crash" reasons are. I saw `CrashLoopBackOff` as a reason, so figured I would just leave it at that.

Reply:
Agree. I dont actually know why we picked this one specific reason to be a metric. I dont see this specific metric in KSM either.

Reply:
Maybe worth bringing up to the broader group, but until there's clarity on that I intend on doing nothing more complex than this.